config/deploy.rb

```diff
@@ -94,3 +95,16 @@ def deploysecret(key)
     end
   end
 end
+
+desc "Convert files which were unicorn sockets into symbolic links to the puma socket, \
+      so legacy nginx configurations pointing to the unicorn socket keep working"
+task :replace_unicorn_with_puma do
+  on roles(:app) do
+    with rails_env: fetch(:rails_env) do
+      execute "mkdir -p #{shared_path}/tmp/sockets; true"
+      execute "mkdir -p #{shared_path}/tmp/pids; true"
+      execute "ln -sf #{shared_path}/tmp/sockets/puma.sock #{shared_path}/sockets/unicorn.sock; true"
```

Comment:
This will give a warning while deploying if the folder `#{shared_path}/sockets` does not exist :thinking:.